apps/sap-commerce-cloud/app-actions/src/actions/fetchBaseSites.ts

```diff
@@ -0,0 +1,41 @@
+import { AppActionCallContext } from '@contentful/node-apps-toolkit';
+import get from 'lodash.get';
+import { Hash } from '../types';
+
+interface AppActionCallParameters {
+  apiKey: string;
+  sapApiEndpoint: string;
+}
+
+export const handler = async (payload: AppActionCallParameters, context: AppActionCallContext) => {
+  const { sapApiEndpoint, apiKey } = payload;
+  try {
+    const req = await fetch(sapApiEndpoint, {
+      method: 'GET',
+      headers: {
+        'Content-Type': 'application/json',
+        'application-interface-key': apiKey,
+      },
+    });
+
+    const res = await req.json();
+
+    return {
+      status: 'Success',
+      // @ts-ignore
```

Comment:
You'll see this in a few places in these app actions. I just opted to ignore ts errors for now since these will need to be refactored to work anyway.